GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
```

Comment:
Group - the classification of a number of persons within an organization; Examples: operations, engineering, sales. It is possible to have groups within groups. Example: the Product and Devops groups can belong to the engineering group. https://github.com/operable/cog/wiki/A-Tour-Through-Cog%27s-Authorization-System#groups